src/foam/u2/view/CurrencyInputView.js

```diff
@@ -86,6 +90,10 @@ foam.CLASS({
       name: 'currency',
       documentation: 'The actual currency object that can format the value'
     },
+    {
+      name: 'shortenToPrecision',
+      documentation: 'Expects an integer as value. Used for RO mode'
+    },
```

Comment:
do we need a default?

Reply:
Nope. Not having a value would just mean the displayed version of the value wont be shortened.